canvas/migrations/0010_team_teamregistration.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.0.14 on 2021-07-12 18:06
```

Comment:
This and the above migration should be combined into a single one. When creating new features want just a single migration, that includes all the changes to the models.